### PR TITLE
[PT-BR] tsconfig.json.md

### PR DESCRIPTION
**Pull request by J-Keven**
@khaosdoctor, @alvarocamillont, @danilofuchs, @orta
- Making corrections
 PR #1069

### CODE REVIEW

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: tsconfig.json
+layout: docs
+permalink: /docs/handbook/tsconfig-json.html
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
permalink: /pt/docs/handbook/tsconfig-json.html
```
The CI is failing due to the missing prefix

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+oneline: Learn about how a TSConfig works
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
oneline: Aprenda sobre como o TSConfig funciona
```

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+translatable: true
+---
+
+## Overview
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
## Visão geral
```

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+A presença de um arquivo `tsconfig.json` em um diretório, indica que esse diretório é a raiz do projeto Typescript.
+O arquivo `tsconfig.json` especifica os arquivos raiz e as configurações necessárias para o compilador compilar o projeto.
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
O arquivo `tsconfig.json` especifica os arquivos raiz e as configurações de compilação necessárias para o projeto.
```
Avoid repetition

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "**/*.spec.ts"]
+}
+```
+
+## TSConfig Básicas
+
+Dependendo do ambiente de execução JavaScript, no qual você deseja executar seu código, pode haver uma configuração básica que pode ser usada em [github.com/tsconfig/bases](https://github.com/tsconfig/bases/).
+Esses são arquivos `tsconfig.json` que seu projeto se estende, o que simplifica o seu `tsconfig.json` ao lidar com o suporte de tempo de execução.
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
Esses são arquivos `tsconfig.json` que seu projeto pode estender, o que simplifica o seu `tsconfig.json` ao já lidarem com o suporte de tempo de execução.
```

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+- [Recommended](https://www.npmjs.com/package/@tsconfig/recommended)
+- [Node 10](https://www.npmjs.com/package/@tsconfig/node10)
+- [Node 12](https://www.npmjs.com/package/@tsconfig/node12)
+- [Deno](https://www.npmjs.com/package/@tsconfig/deno)
+- [React Native](https://www.npmjs.com/package/@tsconfig/react-native)
+- [Svelte](https://www.npmjs.com/package/@tsconfig/svelte)
+
+## Detalhes
+
+A propriedade `"compilerOptions"` pode ser omitida, casos em que os padrões do compilador são usados. Veja nossa lista completa das [Opções que o compilador suporta](/tsconfig).
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
A propriedade `"compilerOptions"` pode ser omitida, casos em que os padrões do compilador são usados. Veja nossa lista completa das [Opções que o compilador suporta](/pt/tsconfig).
```
Take users to the correct language

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+## Referências TSConfig
+
+Você pode saber mais sobre as centenas de opções de configurações em [Referências TSConfig](/tsconfig).
```
Comment (danilofuchs) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
Você pode saber mais sobre as centenas de opções de configurações em [Referências TSConfig](/pt/tsconfig).
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]
Great job!
This is a very important page and we must be very careful with this translation in particular.
Overall, an awesome translation! I only found some minor issues/missing bits.

File: packages/documentation/copy/pt/project-config/tsconfig.json.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+    "preserveConstEnums": true,
+    "outFile": "../../built/local/tsc.js",
+    "sourceMap": true
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "**/*.spec.ts"]
+}
+```
+
+## TSConfig Básicas
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/project-config/tsconfig.json.md]:
```suggestion
## Configurações Básicas
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
One small change

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**